.github/scripts/install_slices/install_slices.py

```diff
@@ -128,9 +128,10 @@ def parse_archive(release: str) -> Archive:
         sys.exit(1)
     # load the yaml data into Archive
     archive_data = data["archives"]["ubuntu"]
-    archive = Archive(
-        str(archive_data["version"]), archive_data["components"], archive_data["suites"]
-    )
+    version = archive_data["version"]
+    if isinstance(version, float):
```

Comment:
is there ever a time where this won't be a float?

Reply:
> is there ever a time where this won't be a float?

I think that may happen if something encloses the version with quotes.